### PR TITLE
Meeting summaries for 2016-09-22

### PR DESCRIPTION
**Pull request by G1lius**

### CODE REVIEW

**Comment by G1lius** [comment]
For some reason the list for 0.13.1 tagged PR's doesn't show correctly on the site. It does show correctly in the preview here.

**Comment by MarcoFalke** [comment]
@G1lius Maybe the list items can not start with a square bracket?

File: _posts/en/meetings/2016-09-22-meeting.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+In [last weeks meeting](/en/meetings/2016/09/15/#release) developers decided to drop PR [#8635][] (Enforce mandatory softfork flags for segwit block/tx) from the [0.13.1 milestone](https://github.com/bitcoin/bitcoin/milestone/22) tag.
+
+### meeting comments
+
+Jl2012 suggests not to do PR [#8654][] (Reuse sighash computations across evaluation) in 0.13.1. Everyone seems to agree.
+There is some disagreement on the RPC behavior change in PR [#8757][] (Fix issue with conflicted mempool tx in listsinceblock). Since people probably built apps on the "strange" behavior it would make more sense to do this in a major release where it's fully documented in the release notes.
+
+Updates on the remaining pull requests tagged for 0.13.1: 
+- PR [#8634][] (Add policy: null signature for failed CHECK(MULTI)SIG) has a lot of ACKs and seems ready to merge.
```
Comment (btcdrak) [inline _posts/en/meetings/2016-09-22-meeting.md]: I think it needs a blank line before the list starts

File: _posts/en/meetings/2016-09-22-meeting.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+Bitcoin Core 0.13.0 is [released](/en/2016/08/23/release-0.13.0/) on 2016/08/23. The next point release 0.13.1 will probably include the segregated witness softfork activation logic, among other bugfixes and optimizations.
+
+In [last weeks meeting](/en/meetings/2016/09/15/#release) developers decided to drop PR [#8635][] (Enforce mandatory softfork flags for segwit block/tx) from the [0.13.1 milestone](https://github.com/bitcoin/bitcoin/milestone/22) tag.
+
+### meeting comments
+
+Jl2012 suggests not to do PR [#8654][] (Reuse sighash computations across evaluation) in 0.13.1. Everyone seems to agree.
+There is some disagreement on the RPC behavior change in PR [#8757][] (Fix issue with conflicted mempool tx in listsinceblock). Since people probably built apps on the "strange" behavior it would make more sense to do this in a major release where it's fully documented in the release notes.
+
+Updates on the remaining pull requests tagged for 0.13.1:
```
Comment (btcdrak) [inline _posts/en/meetings/2016-09-22-meeting.md]: I think it needs a blank line before the list starts

File: _posts/en/meetings/2016-09-22-meeting.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+### background
+
+The bitcoin alert system was a way for trusted parties to broadcast a message to all Core clients about critical network problems. It has been removed in Bitcoin Core since 0.13.0 and has been disabled for a while now.
+
+Earlier this month Gmaxwell [posted](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2016-September/013104.html) on the mailinglist the desire to complete the retirement of the alert system, which has been well received.
+
+### meeting comments
+
+Gmaxwell proposed these next steps:
```
Comment (btcdrak) [inline _posts/en/meetings/2016-09-22-meeting.md]: Again, blank line here before you start the list.

**Pull request merged**